dev.env

```diff
@@ -73,6 +73,17 @@ PATH=$(prepend_path "$PATH" "$VTROOT/dist/chromedriver")
 PATH=$(prepend_path "$PATH" "$VTROOT/dist/node/bin")
 export PATH
 
+# Etcd path.
+case $(uname) in
+  Linux)  etcd_platform=linux;;
+  Darwin) etcd_platform=darwin;;
+esac
+
+ETCD_VERSION=$(cat "${VTROOT}/dist/etcd/.installed_version")
+ETCD_BINDIR="${VTROOT}/dist/etcd/etcd-${ETCD_VERSION}-${etcd_platform}-amd64/"
+PATH=$(prepend_path "$PATH" "$ETCD_BINDIR")
+export PATH
```

Comment:
I have a competing patch, which is to move etcd to a preinstall dependency: https://github.com/vitessio/vitess/pull/5365

It looks like no other part of your patch refers to this, so assuming that my patch merges we should be able to remove these lines.